### PR TITLE
ex 5 and 6 done

### PR DESCRIPTION
**Pull request by vinc86**

### CODE REVIEW

**Comment by vinc86** [comment]
added ex 6 solution

File: 06-PB-ternary-exercises/index.js [hunk 1]
```diff
@@ -24,6 +24,9 @@ console.log(isStudent ? "Ticket costs €5,00" : "Ticket costs €12,00");
...
 
-let okMarie = null;
-let cake = false;
-console.log(cake ? (okMarie = "Let them eat cake") : (okMarie = "Oh, bother"));
+/* let okMarie = null; */
+let cake = true;
+/* console.log(cake ? (okMarie = "Let them eat cake") : (okMarie = "Oh, bother")); */
+
+let okMarie = cake === true ? "Let them eat cake" : "Oh, bother";
+console.log(okMarie);
```
Comment (martl) [inline 06-PB-ternary-exercises/index.js:32]: perfect!

File: 06-PB-ternary-exercises/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// task 1
```
[the remark this answers is not in the document]
Comment (vinc86) [reply 06-PB-ternary-exercises/index.js:3]: added!

File: 05-PB-operators-comparison/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+// task 0
```
[the remark this answers is not in the document]
Comment (vinc86) [reply 05-PB-operators-comparison/index.js:3]: added!

File: 06-PB-ternary-exercises/index.js (hunk 1 above)
Comment (vinc86) [reply 06-PB-ternary-exercises/index.js:34]: Thank you!

**Pull request closed without merging**